code-reviews/tools.md

```diff
@@ -0,0 +1,34 @@
+# Code review tools
+
+## Visual Studio Code
+
+### GitHub: [GitHub Pull Requests](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github)
+
+Supports GitHub pull request processing inside of VS Code.
+
+1. Open the plugin from the "Activity Bar"
```

Comment:
Matter of taste and just a thought: When referring to items in menus etc. rather than using quotation marks, one could use different font style (italic/bold)

Reply:
I don't really mind. We should try to be consistent. Let me try to find some other occurences.

Reply:
Azure docs is using bold as an indication for Menus/Links. I think I'll go for that. Thanks.